### ISSUE TITLE
Heroic calls PowerShell, which uses $PROFILE, which can contain commands that break Heroic (like oh-my-posh)

### ISSUE THREAD

**Issue by ehawman** [Heroic-Games-Launcher/HeroicGamesLauncher#4227]
### Describe the bug

TL;DR: Heroic is using PowerShell on Windows, which calls a user's $PROFILE, and some things (like oh-my-posh) can interfere with Heroic. Please use `-NoProfile` to prevent this.

Installed Heroic. Tried to install a game. "Couldn't get game info". Logs had errors, but ALSO had the output of `fastfetch` over and over. This was weird to me, so I went to my $PROFILE and commented it out. Heroic instantly started working. I went line-by-line until I identified `oh-my-posh --init --shell pwsh --config $themePath | Invoke-Expression` as the culprit. After discussing it with Discord user `arieljuod` in thread "oh-my-posh causes SyntaxError: Unexpected non-whitespace character after JSON" (which in hindsight is an AWFUL title), it was determined that passing the `-NoProfile` flag when calling pwsh would likely avert this issue.

### Add logs

```text
Heroic | https://0x0.st/8sD5.log
Legendary | https://0x0.st/8sDk.log

I had to redact these logs since fastfetch was dumping all my info.
```

### Steps to reproduce

1. Have a Powershell profile with an offending command (`oh-my-posh` in my case, but presumably other commands could cause problems)
2. Try to use Heroic
3. Cry

### Expected behavior

Heroic shouldn't be tripped up by people's pwsh profiles. Hopefully `-NoProfile` is enough to avert the issue.

### Heroic Version

Latest Stable

### System Information

- OS [e. g. "Ubuntu"]: Windows 111

**Comment by arielj** [comment]
I just noticed this is a duplicate of https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/3828, closing in favor of that one that was created before
